### ISSUE TITLE
Tagged decorators wont accept two interpolated strings  

### ISSUE THREAD

**Issue by hareeqi** [tizmagik/react-csjs#26]
only in the tagged decorators if you interpolating more than one string in the style only the first one get interpolated while the second one always return empty string (In the example below color2 never appears and instead return empty string )

``` javascript
import React from 'react';
import csjs from 'react-csjs';

const color1 = '#00ff00';
const color2= '#ff0000';

@csjs`
.button {
  background-color: ${color1};
  color: ${color2};
}`
export default class Button extends React.Component {
  render() {
    return (
      <div className={this.props.classes.button}>
         {'Hello World'}
      </div>
    )
  }
}
```

while if you use Default export wrapper everything works fine and both color1 and color2 appears.

``` javascript
import React from 'react';
import csjs from 'csjs';
import withStyles from 'react-csjs';

const color1 = '#00ff00';
const color2= '#ff0000';

const style = csjs`
.button {
  background-color: ${color1};
  color: ${color2};
}`
const Button  = ({classes}) =>  (
      <div className={classes.button}>
         {'Hello World'}
      </div>
    )

export default withStyles(style)(Button)
```

This is important if you have a theme file and you are importing it every where.

**Comment by tizmagik** [comment]
Ah, thanks for the bug report @hareeqi ! I think this is an oversight on our part and should be a quick fix.

**Comment by tizmagik** [comment]
cc @drcmda

**Comment by tizmagik** [comment]
Fixed with v1.3.1 @hareeqi
